tests/psalm-baseline.xml

```diff
@@ -26,6 +29,17 @@
       <code><![CDATA[$this->request->server]]></code>
     </NoInterfaceProperties>
   </file>
+  <file src="lib/Federation/CloudFederationProviderTalk.php">
+    <UndefinedClass>
+      <code><![CDATA[ISignedCloudFederationProvider]]></code>
```

Comment:
Should be not needed anymore after https://github.com/nextcloud-deps/ocp/pull/28 is there